proto/pipeline.proto

```diff
@@ -58,8 +58,9 @@ message Dispatch {
   repeated WrapNode remote_connector = 3;
   repeated int32 ShuffleRegIdxLocal=4;
   repeated int32 ShuffleRegIdxRemote=5;
-  bool  is_sink = 6;
-  bool  rec_sink = 7;
+  int32 ShuffleType=6;
```

Comment:
variable name in protobuf should use snake case

Reply:
fixed